komodo/cli.py

```diff
@@ -103,10 +103,10 @@ def _main(args):
         _print_timings(timings[-1])
 
         shell(f"mv {args.release + str(tmp_prefix)} {args.release}")
-        shell(
-            "rmdir -p --ignore-fail-on-non-empty "
-            f"{args.release + str(tmp_prefix.parent)}"
-        )
+        try:
```

Comment:
This change should be in a separate commit, as it is motivated by mac os compatibility

(when we later look at the blame for this, we will not understand why we claimed that this was for input validation)

Reply:
This was in the "Fix cli unit test not running on mac"-commit. Should I put it in another PR?
The commit name might be misleading, what would be a more suitable title?